pages/state/useTodoState.tsx

```diff
@@ -0,0 +1,55 @@
+import { useState } from 'react'
+import { v4 as uuidv4 } from 'uuid';
+
+ type todo = Array<{id:String, todo:string, isDone:boolean, isEdit:boolean}>
```

Comment:
not use contracture Array or String use type `string` or `[]`

```suggestion
 type todo = { id:string, todo:string, isDone:boolean, isEdit:boolean}[]
```